frontend/README.md

```diff
@@ -2,33 +2,6 @@
 
 This is the frontend, which consumes the exposed Trubudget API.
```

Comment:
the section [`Production vs. Development Mode`](https://github.com/openkfw/TruBudget/blob/c3a0c7daefa3d12ed73b78880340d400352ef57e/frontend/README.md#production-vs-development-mode) is a little out of date:

- [ ] "It is possible to check the mode inside the Javascript Code by checking process.env.NODE_NEV with will return development or production depending on the mode." - this env var was removed in 2.0. The only difference between 
- [ ] the `./startDev`script does not exist anymore, no dummy backend is spinened up
- [ ] no `DOCKER_PASSWORD`is used anymore

maybe other things are also out of date, so double check please